custom_components/mila/__init__.py

```diff
@@ -78,8 +78,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     conf_devices = options.get(CONF_DEVICES, data[CONF_DEVICES])
     conf_identifiers = [(DOMAIN, resource_id) for resource_id in conf_devices]
 
-    device_registry = await hass.helpers.device_registry.async_get_registry()
-    # entity_registry = await hass.helpers.entity_registry.async_get_registry()
+    device_registry = hass.helpers.device_registry.async_get(hass)
```

Comment:
Don't we need `await` here?

Reply:
I thought so but is throws a error when I add await

Reply:
this is the error:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 339, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/mila/__init__.py", line 81, in async_setup_entry
    device_registry = await hass.helpers.device_registry.async_get(hass)
TypeError: object DeviceRegistry can't be used in 'await' expression